src/Lazy/Lazy/UnsafeLazy.cs

```diff
@@ -0,0 +1,36 @@
+namespace Lazy.Lazy;
+
+/// <summary>
+/// Lazy evaluation class for single-threaded execution.
+/// <inheritdoc cref="Lazy{T}"/>
+/// </summary>
+/// <typeparam name="T">Result type of lazy computed expression, see <see cref="Lazy{T}"/></typeparam>
+public class UnsafeLazy<T> : Lazy<T>
+{
+
+    /// <summary>
```

Comment:
```suggestion
{
    /// <summary>
```
После { пустая строка не ставится